### PR TITLE
JBPM-7473: Stunner - New Marshallers for Conditional events

### PR DESCRIPTION
**Pull request by wmedvede**

### CODE REVIEW

**Comment by wmedvede** [comment]
Hello  @evacchi @hasys would you mind review this PR?

p.s. some prioritization will be much appreciated since I leave for PTO this Friday.

Thanks!

**Comment by wmedvede** [comment]
Jenkins execute full downstream build

**Comment by wmedvede** [comment]
Current repo build failing due to:
org.kie.workbench.common.services.backend.compiler.DefaultMavenCompilerTest.buildWithPullRebaseUberfireTest
org.kie.workbench.common.services.backend.compiler.kie.KieDefaultMavenCompilerTest.buildCompileWithOverrideOnRegularFSTest

Let's see wait for the full-build. If it fails I'll rebase with master and update as suggested by @jhrcek

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-backend/src/main/java/org/kie/workbench/common/stunner/bpmn/backend/converters/fromstunner/properties/EventPropertyWriter.java [hunk 1]
```diff
@@ -151,5 +156,22 @@ public void addTimer(TimerSettings timerSettings) {
...
+        ConditionalEventDefinition conditionalEventDefinition = bpmn2.createConditionalEventDefinition();
+        FormalExpression conditionExpression = bpmn2.createFormalExpression();
+
+        String languageFormat = Scripts.scriptLanguageToUri(condition.getValue().getLanguage(),
+                                                            Scripts.LANGUAGE.DROOLS.format());
+        conditionExpression.setLanguage(languageFormat);
+
+        String conditionScript = condition.getValue().getScript();
+        if (conditionScript != null && !conditionScript.isEmpty()) {
+            conditionExpression.setBody(asCData(StringEscapeUtils.escapeXml10(conditionScript)));
```
Comment (evacchi) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-backend/src/main/java/org/kie/workbench/common/stunner/bpmn/backend/converters/fromstunner/properties/EventPropertyWriter.java]: this is already wrapped in CData, I don't think escaping is needed

**Comment by wmedvede** [comment]
@evacchi suggestion included, thanks!

for the record: also rebased with master

**Comment by hasys** [comment]
Jenkins retest please

**Pull request merged**